plasmapy/particles/tests/test_particle_class.py

```diff
@@ -997,3 +997,36 @@ def test_customized_particle_repr(cls, kwargs, expected_repr):
             f"from_str: {from_str}"
             f"from_repr: {from_repr}"
         )
+
+
+particle_json_repr_table = [
+    (
+        Particle,
+        {"argument": 'lead'},
+        '{"type": "Particle", "symbol": "Pb"}',
+    ),
+    (
+        CustomParticle,
+        {"mass": 5.12 * u.kg, "charge": 6.2 * u.C},
+        '{"type": "CustomParticle", "mass": {"quantity": 5.12, "unit": "kg"}, "charge": {"quantity": 6.2, "unit": "C"}}',
+    ),
+    (
+        DimensionlessParticle,
+        {"mass": 5.2, "charge": 6.3},
+        '{"type": "DimensionlessParticle", "mass": {"quantity": 5.2}, "charge": {"quantity": 6.3}}',
+    ),
+]
+
+
+@pytest.mark.parametrize("cls, kwargs, expected_repr", particle_json_repr_table)
+def test_customized_particle_repr(cls, kwargs, expected_repr):
+    """Test the JSON representations of normal, dimensionless and custom particles."""
+    instance = cls(**kwargs)
+    json_repr = instance.to_json()
+    if not expected_repr == json_repr:
```

Comment:
I think the standard pytest idiom would be `assert expected_repr == json_repr, "optional_description:`.

Reply:
Ah I see! Thanks for the tip. I'll make that change :)